salt/minion.py

```diff
@@ -1881,6 +1881,9 @@ def handle_schedule():
     def _handle_payload(self, payload):
         if payload is not None and self._target_load(payload['load']):
             self._handle_decoded_payload(payload['load'])
+        elif self.opts['zmq_filtering']:
```

Comment:
IMO if you want this debug line it should only print if `_target_load` returns False-- otherwise payload of None would also trigger this-- which we probably don't care about. Also, without any information this line might not be terribly helpful for debugging. Maybe this should me moved to trace and include the payload?